javascript/src/test/utils/BasicCourseTableHelpers.test.js

```diff
@@ -290,4 +978,337 @@ const sectionsOutput = [
+  //     "courseId": 'CMPSC     8  ',          
+  //     "title": 'INTRO TO COMP SCI',         
+  //     "unitsFixed": 4                       
+  //   }                                     
+  // },                                      
+  // {                                       
+  //   "enrollCode": '07526',                  
+  //   "section": '0103',                      
+  //   "classClosed": null,                    
+  //   "courseCancelled": null,
+  //   "enrolledTotal": 25,                    
+  //   "maxEnroll": 25,                        
+  //   "timeLocations": [
+  //     {
+  //       "room": "LINE",
+  //       "building": "ON",
+  //       "roomCapacity": 999,
+  //       "days": "  W    ",
+  //       "beginTime": "11:00",
+  //       "endTime": "11:50"
+  //     }
+  //   ],          
+  //   "instructors": [],                      
+  //   "course": {                             
+  //     "courseId": 'CMPSC     8  ',          
+  //     "title": 'INTRO TO COMP SCI',         
+  //     "unitsFixed": 4                       
+  //   }                                     
+  // },                                      
+  {                                       
+    "enrollCode": '07534',                  
+    "section": '0104',                      
+    "classClosed": null,                    
+    "courseCancelled": null,
+    "enrolledTotal": 25,                    
+    "maxEnroll": 25,                        
+    "timeLocations": [
+      {
+        "room": "LINE",
+        "building": "ON",
+        "roomCapacity": 999,
+        "days": "  W    ",
+        "beginTime": "12:00",
+        "endTime": "12:50"
+      }
+    ],          
+    "instructors": [],                      
+    "course": {                             
+      "courseId": 'CMPSC     8  ',          
+      "title": 'INTRO TO COMP SCI',         
+      "unitsFixed": 4                       
+    }                                     
+  },                                      
+  // {                                       
+  //   "enrollCode": '63404',                  
+  //   "section": '0105',                      
+  //   "classClosed": 'Y',                     
+  //   "courseCancelled": null,                              
+  //   "enrolledTotal": null,                  
+  //   "maxEnroll": 25,                        
+  //   "timeLocations": [
+  //     {
+  //       "room": "LINE",
+  //       "building": "ON",
+  //       "roomCapacity": 999,
+  //       "days": "  W    ",
+  //       "beginTime": "13:00",
+  //       "endTime": "13:50"
+  //     }
+  //   ],          
+  //   "instructors": [],                      
+  //   "course": {                             
+  //     "courseId": 'CMPSC     8  ',          
+  //     "title": 'INTRO TO COMP SCI',         
+  //     "unitsFixed": 4                       
+  //   }                                     
+  // }                                       
```

Comment:
What is all of this commented out code doing here?  

Remove it if it's not needed.   We avoid merging commented out code into the `main` branch.